src/components/dashboard/HeaderDashboard.js

```diff
@@ -42,7 +42,7 @@ export const HeaderDashboard = () => {
                 <Navbar.Collapse id="responsive-navbar-nav">
                     <Nav className="navbar-custom">
                         <Nav.Link
-                            href="#home"
+                            href="https://zinzen.vercel.app/Home"
```

Comment:
This seems like the code that would resolve #148. Please make sure to target only one issue per PR or at least mention explicitly when you are working on more than one issue within the same PR. I'm going to approve this for now but follow this convention moving forward